### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by hroncok**

### CODE REVIEW

File: python_versions_check.py [hunk 1]
```diff
@@ -31,17 +32,22 @@
...
-    results_path = os.path.join(artifactsdir, 'taskotron', 'results.yml')
-    artifact = os.path.join(artifactsdir, 'output.log')
+    artifactsdir = pathlib.Path(artifactsdir)
+    workdir = pathlib.Path(workdir).resolve()
+    resultsdir = artifactsdir / 'taskotron'
+    resultspath = resultsdir / 'results.yml'
+    artifact = artifactsdir / 'output.log'
+
+    artifactsdir.mkdir(parents=True, exist_ok=True)
+    resultsdir.mkdir(parents=True, exist_ok=True)
```
Comment (hroncok) [inline python_versions_check.py]: Those two mkdirs were needed.
cc @kparal and @frantisekz - previously it worked without this

File: test/integration/test_integration.py [hunk 2]
```diff
@@ -177,6 +178,9 @@ def results(request):
 _pycallgraph = fixtures_factory('python-pycallgraph-0.5.1-13.fc28')
 pycallgraph = fixtures_factory('_pycallgraph')
 
+_ttystatus = fixtures_factory('python-ttystatus-0.34-7.fc29')
+ttystatus = fixtures_factory('_ttystatus')
```
Comment (hroncok) [inline test/integration/test_integration.py]: Note that I'd rather replace pycallgraph with another package, but i couldn't find anything that would fit for both usecases  pycallgraph had.

**Comment by encukou** [comment]
The changes look OK to me, but I'm not very familiar with the code.

**Comment by hroncok** [comment]
Thanks @irushchyshyn, @encukou!

**Pull request merged**